nemo/lightning/pytorch/strategies.py

```diff
@@ -45,6 +46,18 @@
 DDPLiteral = Literal["megatron", "pytorch"]
 
 
+@dataclass
+class ParallelismConfig:
```

Comment:
Do we even need the defaults? Since they are always being set by the MegatronStrategy

Reply:
sure let me drop these.